.cargo/config

```diff
@@ -1,2 +1,6 @@
 [net]
     git-fetch-with-cli = true
+
+# Statically compile MSVC runtime
+[target.'cfg(target_env = "msvc")']
+    rustflags = ["-Ctarget-feature=+crt-static"]
```

Comment:
How much does this bloat the size of the binary? Are there any license restrictions that this imposes?

Reply:
Either "pretty much not at all" or it didn't actually work...

Reply:
Fwiw wasmtime does the same: https://github.com/bytecodealliance/wasmtime/blob/08b7c877931a3da4cb73bd3c0961fd475c0e8a60/.github/actions/binary-compatible-builds/main.js#L20